x/protocolpool/keeper/keeper.go

```diff
@@ -229,12 +227,14 @@ func (k Keeper) IterateAndUpdateFundsDistribution(ctx context.Context) error {
 	}
 
 	// next we iterate over the distributions, calculate each recipient's share and the remaining pool funds
-	toDistribute := map[string]math.Int{}
-	poolFunds := math.ZeroInt()
-	fullAmountToDistribute := math.ZeroInt()
+	toDistribute := map[string]sdk.Coins{}
```

Comment:
naming is hard
personal preference: how about `distributeToRecipient` ?

Reply:
accepting these changes 👌